docs/installation.rst

```diff
@@ -14,6 +14,7 @@ Prerequisites
 * If you are new to Mozilla or the A-Team, read the `A-Team Bootcamp`_.
 * Install Git_, Virtualbox_ and Vagrant_ (latest versions recommended).
 * Clone the `treeherder repo`_ from Github.
+* Linux only: An nfsd server is required. You can install this on Ubuntu by running `apt-get install nfs-common nfs-kernel-server`
```

Comment:
Maybe we should link to the troubleshooting section at the end of the "setting up vagrant" section. http://treeherder.readthedocs.io/installation.html#setting-up-vagrant